FrontEnd/src/components/basics/Fallback/Fallback.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from 'react';
+import { amenityToIcon, Amenity } from '../Amenities/Amenities';
+import { Icon as IconType, miscIcons } from '@icons';
+
+// Amenity needs to be any so it can be type guarded
+interface FallbackProps {
+  amenity: any;
+}
+
+// Type guard to check if key is of Amenity type
```

Comment:
nit: the best syntax for header comments for functions/classes is:
`[name here]` is ....
so in this case: `isAmenity is a type guard to...`